Makefile

```diff
@@ -410,7 +413,7 @@ proto-swagger-gen:
 
 proto-format:
 	@echo "Formatting Protobuf files"
-	$(protoImage) find ./ -name *.proto -exec clang-format -i {} \;
+	$(protoImage) buf format --write
```

Comment:
quote: https://buf.build/blog/introducing-buf-format
> Zero configuration
The buf format command has no configuration options. There's only one way to format .proto files, so that every .proto file looks and feels the same way. Stop wasting time and energy on deciding how .proto files ought to be formatted - buf decides for you so you don't have to.

Reply:
node config files needed @coderabbitai